install-deps.sh

```diff
@@ -80,7 +81,7 @@ else
     debian|ubuntu|devuan)
         echo "Using apt-get to install dependencies"
         $SUDO apt-get install -y lsb-release devscripts equivs
-        $SUDO apt-get install -y dpkg-dev gcc
+        $SUDO apt-get install -y dpkg-dev gcc libcunit1-dev
```

Comment:
@liupan1111 if the unittest in spdk compilation is inevitable. can we add it to `debian/control`?

Reply:
@liupan1111 this is not addressed.

Reply:
let me have a try.

Reply:
@tchaikov sorry not understand it clearly, libcunit1-dev is under the control of "     debian|ubuntu|devuan)", what else do you want me to do?